shared/src/packages/users/libs/types/customer-sign-up-response-dto.type.ts

```diff
@@ -1,8 +1,9 @@
 import { type UserEntityT } from './user-entity.type.js';
+import { type UserGroupEntityObjectT } from './user-group-entity.type.js';
 
-type CustomerSignUpResponseDto = Pick<
+type CustomerSignUpResponseDto = Omit<
   UserEntityT,
-  'id' | 'phone' | 'accessToken'
->;
+  'passwordHash' | 'passwordSalt' | 'groupId'
+> & { group: UserGroupEntityObjectT };
```

Comment:
Aren't this duplicate of UserEntityObjectWithGroup? can we declare it like `type CustomerSignUpResponseDto = UserEntityObjectWithGroup`